src/screens/FeedScreen/RenderList.js

```diff
@@ -65,6 +67,23 @@ const RenderListFeed = (props) => {
   } = useFeed();
   const {handleUserName} = useWriteComment();
 
+  const postApiUpvote = async (status) => {
+    await onPressUpvote({
+      activity_id: item.id,
+      status,
+      feed_group: 'main_feed',
+      voteStatus
+    });
+  };
+  const postApiDownvote = async (status) => {
+    await onPressDownVote({
+      activity_id: item.id,
+      status,
+      feed_group: 'main_feed',
+      voteStatus
+    });
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The functions `postApiUpvote` and `postApiDownvote` are defined but not used within the scope of this hunk. If these functions are not used elsewhere in the code, consider removing them to improve readability and maintainability.



<!-- This is an auto-generated comment by OSS CodeRabbit -->